README.md

```diff
@@ -16,66 +16,69 @@ This repository is for Browser.html (the front-end). Active development of Graph
 
-``` sh
-./mach run -r -- -b -w --pref dom.mozbrowser.enabled --pref dom.forcetouch.enabled --pref shell.builtin-key-shortcuts.enabled=false http://localhost:6060
+```sh
+npm run live-server
 ```
 
+### Client
 
-### Running in Gecko
+In order to run browser.html application itself, you will need a client component. This would be a Graphene runtime, built into a servo binary. You can either [download][servo download] pre-built nightly snapshots or [build it yourself][bulid servo] and run with `--browserhtml` flag. Assuming you have pre-built snapshot in default location on a Mac you can run browser.html application with a following command:
 
-Browser.html can also be run on top of a Gecko-based version of Graphene. We sometimes use this variant to test and debug features that haven't yet landed in Servo. Build instructions for Gecko-flavored Graphene can be found [on the wiki](https://github.com/browserhtml/browserhtml/wiki/Building-Graphene-%28Gecko-flavor%29).
 
+``` sh
+ /Applications/Servo.app/Contents/MacOS/servo -b -w --pref dom.mozbrowser.enabled --pref dom.forcetouch.enabled --pref shell.builtin-key-shortcuts.enabled=false http://localhost:6060
+```
 
-### Using WebIDE
-
-The easiest way to use developer tools with Browser.html is to select the "Remote Runtime" option in WebIDE while using the Gecko Graphene runtime.
+### Gecko Client
 
-By default you should be able to connect to the running browser at: localhost:6000.
+Browser.html can also be run on top of a Gecko-based version of Graphene. We used to use this variant to test and debug features that were not yet in Servo. You could either [download][dowload gecko] pre-build nightly snapshots or [build it yourself](https://github.com/browserhtml/browserhtml/wiki/Building-Graphene-%28Gecko-flavor%29). Assuming you have pre-build snapshot in default location on a Mac you can run browser.html appliaction with a following command:
 
+```sh
+/Applications/B2G.app/Contents/MacOS/b2g-bin --start-manifest=http://localhost:6060/manifest.webapp --profile ./.profile
+```
 
-### Integration Tests
+### Electron Client
 
-Run integration tests with `./test/runall.sh`. You need to have a Graphene Gecko binary symlinked in the root of the repository.
+Browser.html can also run as an [Electron][] application. Assuming you have `electron` installed you could run browser.html application with a following command (must be run from the project root):
 
-``` sh
-ln -s ../gecko/obj-graphene/dist/Graphene.app graphene
+```sh
+electron .
 ```
+
+### Browser Client
+
+You could also just load http://localhost:6060/ in your favourite web browser, but be aware that very few features will work in this case given that application additional API not available for web content. 
```

Comment:
Nit: s/could/can/
Also, "very few" is surprisingly not really accurate. Most stuff works quite well in this configuration. How about "many features with not work because they require APIs not available to web content"?